contact.html

```diff
@@ -223,7 +223,7 @@ <h1 class="wow fadeInUp" data-wow-delay="0.3s">Get in touch</h1>
                             <div class="sent-message">Your message has been sent. Thank you!</div>
                         </div>
 						<div class="col-md-offset-3 col-md-6 col-sm-offset-2 col-sm-8">
-							<input name="submit" type="submit" class="form-control submit" id="submit" value="SEND MESSAGE">
+							<input name="submit" type="button" class="form-control submit" id="submit" value="SEND MESSAGE">
```

Comment:
The type must be "submit".  The real issue in the button that is not visible as the button to the users. You need to work on the CSS only.

Reply:
![image](https://user-images.githubusercontent.com/37388848/142973273-30f83b4c-7aea-40b4-a1fb-c91ad7aef4ef.png)

The button must be look like this and it must have the hover effect.

Reply:
@pankajan05 okay

Reply:
@pankajan05 assign this bug to me i'll fix it

Reply:
@Manoj0Marmat I really appreciate your enthusiasm. Please contact this Email : lilan@elzian.com

Reply:
@pankajan05 i fix it

Reply:
@pankajan05 somehow the background, colour and hover:background not apply properly that's why submit massage button messed up